### PR TITLE
38 readme.md

### PR DESCRIPTION
**Pull request by mmorihiro**
Fix for #38

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,11 +1,19 @@
...
 
-[![Build Status](https://travis-ci.org/mmorihiro/LifeGame.svg?branch=master)](https://travis-ci.org/mmorihiro/LifeGame) [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
+[![Build Status](https://travis-ci.org/mmorihiro/LifeGame.svg?branch=master)](https://travis-ci.org/mmorihiro/LifeGame) [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)  
+##Demo
+![Demo](demo.gif) 
 ##Description
 A simple game of life written with Kotlin and libGDX.
 ##Download
 Not released yet. After finish v0.3, first one will be released.  
+libGDX is a cross platform game library.However, this game only targets desktop(Windows, Mac, Linux) now.
```
Comment (prince-0203) [inline README.md]: A space between "library" and "However" is missing.

File: README.md [hunk 2]
```diff
@@ -1,11 +1,19 @@
...
 Not released yet. After finish v0.3, first one will be released.  
+libGDX is a cross platform game library.However, this game only targets desktop(Windows, Mac, Linux) now.  
 If you want to try the under development version, use `git clone https://github.com/mmorihiro/LifeGame.git`  
 Then, move to the project root and run `./gradlew run`
 ##How to play
-TODO
+Very simple and easy.  
+
+* You can change life or death by clicking the field.
+* This game has just 3 buttons. Press and see how it works.
```
Comment (prince-0203) [inline README.md]: You need a blank line after the list.

File: README.md (hunk 1 above)
Comment (forestone23) [inline README.md]: A space between "library" and "However" is missing.

File: README.md (hunk 2 above)
Comment (forestone23) [inline README.md]: You need a blank line after the list.

Comment (forestone23) [inline README.md:17]: You need a blank line after the list.

File: README.md (hunk 1 above)
Comment (forestone23) [inline README.md]: A space between "library" and "However" is missing.

**Review by forestone23 — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (mmorihiro) [reply README.md:17]: What a big mistake!
I will fix it.

**Review by forestone23 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (mmorihiro) [reply README.md]: I didn't pay attention to that.
Fixed with 3d6b75f

File: README.md (hunk 2 above)
Comment (mmorihiro) [reply README.md:17]: Fixed with 3d6b75f

**Pull request merged**